clang/lib/StaticAnalyzer/Checkers/MallocChecker.cpp

```diff
@@ -1670,15 +1702,18 @@ MallocChecker::MallocMemReturnsAttr(CheckerContext &C, const CallEvent &Call,
   if (!State)
     return nullptr;
 
-  if (Att->getModule()->getName() != "malloc")
-    return nullptr;
+  // Preseve previous behavior when "malloc" class means AF_Malloc
+  auto attrClassName = Att->getModule()->getName();
+  auto AF = attrClassName == "malloc"
+                ? AF_Malloc
+                : AllocationFamily(AF_Custom, attrClassName);
```

Comment:
You could do this in the ctor of `AllocationFamily`. That would ensure that it's used like that consistently across all sites.

I've seen a variable name `Family` for representing the same value before. Have you considered using the same variable name to boost consistency and readability across the patch?